### ISSUE TITLE
The 22 Immutable Laws of Marketing

### ISSUE THREAD

**Issue by quinnliu** [quinnliu/smith_hunter#51]
## Table of contents
- law of leadership
- law of category
- law of the mind
- law of perception
...

**Comment by quinnliu** [comment]
- When a company makes a mistake today, foot-prints quickly show up on its back as competition runs off with its business. To get the businness back, the company has to wait for others to make mistakes and then figure out how to exploit the situation.

**Comment by quinnliu** [comment]
- it's human nature not to admit there are things you can't do. Certainly most marketers believe that anything is achievable if you are energetic enough, or creative enough, or determined enough.

**Comment by quinnliu** [comment]
- The Law of Leadership. It's better to be first than it is to be better.
- The second person to fly the Atlantic Ocean solo was Bert Hinkler. Bert was a better pilot than Charlie - he flew faster, he consumed less fuel. Yet who has ever hard of Bert Hinkler?
- Most companies go the Bert Hinkler route. They wait until a market develops.

**Comment by quinnliu** [comment]
- The leading brand in any category is almost always the first brand into the prospect's mind. Hertz in rent-a-cars. IBM in computers. Coca-Cola in cola.

**Comment by quinnliu** [comment]
- People tend to stick with what they've got. If u meet someone a little better than ur wife or husband, it's really not worth making the switch.
- One reason for the first brand tends to maintain it's leadership is that the name often becomes generic. Xerox, the first plan-paper copier, became the name for all plain-paper copiers.

**Comment by quinnliu** [comment]
- Most people use brand names when they become generic: Band-Aid, Fiberglas, Formica, Gore-Tex, Jello, Krazy Glue, Q-tips, Saran Wrap, Velcro. 
- Marketing is a battle of perceptions, not products.

**Comment by quinnliu** [comment]
# The Law of the Caategory: If u can't be first in a category, set up a new category u can be first in
- This is the largest community of people learning MIPS from a 21 video series :)

**Comment by quinnliu** [comment]
- Commodore was just another manufacturer of home personal computers that wasn't going anywhere until i positioned the Amiga as the first multimedia computer. 
- There are many different ways to be first. Dell got into the crowded personal computer field by being the first to sell computers by phone.

**Comment by quinnliu** [comment]
- Ask urself "First what?"

**Comment by quinnliu** [comment]
# The Law of the Mind: It's better to be first in the mind than to be first in the marketplace

**Comment by quinnliu** [comment]
- thousands of would-be entrepreneurs are tripped up every year by this law. Someone has an idea or concept he or she believes will revolutionize an industry, as well it may. The problem is getting the idea or concept into the prospect's mind.

**Comment by quinnliu** [comment]
- Xerox was first in copiers and then tried to get into the computer business. Twenty-five years and $2 billion later, Xerox is nowhere in computers.
- The single most wasteful thing u can do in marketing is trying to change a mind.

**Comment by quinnliu** [comment]
- if u want to make a big impression on another person, u cannot worm ur way into their mind & then slowly build up a favorable opinion over a period of time. the mind doesn't work that way. u have to blast ur way into the mind.
- the reason u blast instead of worm is that people don't like to change their minds. once they perceive u one way, that's it. they find of file u away in their minds as a certain kind of person. u cannot become a different person in their minds.

**Comment by quinnliu** [comment]
- Apple's problem in getting into it's prospects' minds was helped by its simple, easy-to-remember name. On the other hand, Apple's competitors had complicated names that were difficult to remember. In the early days, five personal computers were in position on the launching pad: Apple II, Commodore Pet, IMSAI 8080, MITS Altair 8800, and Radio Shack TRS-80.

**Comment by quinnliu** [comment]
# The Law of Perception: Marketing is not a battle of products, it's a battle of perceptions.
- There is no objective reality. There are no facts. There are no best products. All that exists in the world of marketing are perceptions in the minds of the customer or prospect. The perception is the reality. Everything else is an illusion.
- most people think they are better perceivers than others. they have a sense of personal infallibility. their perceptions are always more accurate than those of their neighbors or friends. truth & perception become fused in the mind, leaving no difference between the two.

**Comment by quinnliu** [comment]
- People cling firmly to the belief that reality is the world outside of the mind & that the individual is one small speck on a global spaceship. Actually it's the opposite. The only reality u can be sure about is in ur own perceptions. If the universe exists, it exists inside ur own mind and the minds of others.

**Comment by quinnliu** [comment]
- But in Japan, Honda is nowhere near the leader. There, Honda is in third place, behind Toyota and Nissan. Toyota sells more than four times as many automobiles in Japan as Honda does. So what's the difference between Honda in Japan & Honda in the US? The products are the same, but the perceptions in customer's minds are different.
- Marketing is a battle of perceptions, not products. Marketing is the process of dealing with those perceptions.

**Comment by quinnliu** [comment]
- Everybody knows there's a problem with Audi cars. On 11/23/1986, CBS broadcast a 60 minutes segment called "out of control". it called attention to a number of complaints about Audi's "unintended acceleration". American sales of Audis fell through the floorboards - from 60,000 in 1986 to 12,000 in 1991. But have u ever personally had any problems with "unintended acceleration" while test-driving an Audi? It is unlikely. Every single automobile expert who has tested the car has failed to duplicate the complaint. Yet the perception lingers on.

**Comment by quinnliu** [comment]
# The Law of Focus: the most powerful concept in marketing is owning a word in the prospect's mind

**Comment by quinnliu** [comment]
- Federal Express was able to put the word overnight into the minds of its prospects because it sacrificed its product line and focused on overnight package delivery only. In a way, the law of leadership - it's better to be first than to be better - enables the first brand or company to own a word in the mind of the prospect. But the word the leader owns is so simple that it's invisible.

**Comment by quinnliu** [comment]
- U don't have to be a linguistic genius to find a winner. Prego went against leader Ragu in the spaghetti sauce market and captured a 27 percent share with an idea borrowed from Heinz. Prego's word is thicker.
- The most effective words are simple and benefit oriented. No matter how complicated the product, no matter how complicated the needs of the market, it's always better to focus on one word or benefit rather than 2 or 3 or 4. 
- The halo effect. If u strongly establish one benefit, the prospect is likely to give u a lot of other benefits, too. A "thicker" spaghetti sauce implies qualtiy, nourishing ingredients, value...

**Comment by quinnliu** [comment]
Crest = cavities
Mercedes = engineering
BMW = driving
Volvo = safety
Domino's = home delivery
Pepsi-Cola = youth
Nordstrom = service

**Comment by quinnliu** [comment]
- The essence of marketing is narrowing the focus. U become stronger when u reduce the scope of ur operations. u can't stand for something if u chase after everything

**Comment by quinnliu** [comment]
- You'd think the antidrug forces (who, after all, are professionals) would have taken a leaf from the amateurs fighting the abortion issue. Both sides of the abortion issue have focused on single, powerful words - pro-life & pro-choice. 
- One word that could do this is the ultimate down word, loser. Since drug usage causes all kinds of losses (of job, family, self-esteem, freedom, life), a program that said "Drugs are for losers" could have a very powerful impact, especially on the recreational user, who is more concerned with social status than with getting high. 
- the law of focus, a marketing law, could help solve one of society's biggest problem.

**Comment by quinnliu** [comment]
# The law of Exclusivity: two companies cannot own the same word in the prospect's mind

**Comment by quinnliu** [comment]
- What researchers never tell u is that some other company already owns the idea. They would rather encourage clients to mount massive marketing programs. The theory is that if u spend enough money, u can own the idea. Right? Wrong.

**Comment by quinnliu** [comment]
- Some years ago Burger King started down this slippery slope from which it has never quite recorvered. A market study showed that the most popular attribute for fast food was "fast". So Burger King did what most red-blooded marketers do. It turned to its advertising agency and said, "if the world wants fast, our advertising should tell them we're fast".

**Comment by quinnliu** [comment]
# The law of the ladder: the strategy to use depends on which rung u occupy on the lader

**Comment by quinnliu** [comment]
- Then Avis did the one thing u have to do to make progress inside the mind of the prospect. They acknowledged their position on the ladder. "Avis is only No. 2 in rent-a-cars. So why go with us? We try harder." 
- For 13 years in a row, Avis had lost money. Then, when it admitted to being No. 2, it started to make money, lots of money. 
- the mind is selective. prospects use their ladders in deciding which information to accept & which information to reject. in general, a mind accepts only new data that is consistent with its product ladder in that categroy. everything else is ignored.

**Comment by quinnliu** [comment]
- How many rungs are there on ur ladder? It depends on whether ur product is a high-interest or a low-interest product. Products u use every day tend to be high-interest products with many rungs on their ladders. Products that are purchased infrequently usually have few rungs on their ladders.
- products that involve a great deal of personal pride (automobiles, watches, cameras) are also high-interest products with many rungs on their ladders even though they are purchased infrequently.

**Comment by quinnliu** [comment]
- Products that are purchased infrequently and involve an unpleasant experience usually have very few rungs on their ladders. Automobile batteries, tires, and life insurance are three examples.
- the, ultimate product that involves the least amount of pleasure and is purchased once in a lifetime has no rungs on its ladder. Ever hear of Batesville caskets? Probably not, although the brand has almost 50 percent of the market.

**Comment by quinnliu** [comment]
- Marketing people often talk about the "three leading brands" in a category as if it were a battle of equals. It almost never is. The leader inevitably dominates the no. 2 brand and the no. 2 brand inevitably smothers no. 3. In baby food, it's Gerber, Beech-nut, and Heinz. In bear, it's budweiser, miller, and coors. In long-distance telephone service it's AT&T, MCI, & Sprint.

**Comment by quinnliu** [comment]
- According to Harvard psychologist Dr. George A. Miller, the avg. human mind cannot deal with more than seven units at a time.
- I disagree with better to be No. 3 on a big ladder than No. 1 on a small ladder

**Comment by quinnliu** [comment]
# The law of duality: in the long run, every market becomes a two-horse race

**Comment by quinnliu** [comment]
- in batteries, it's Eveready and Duracell. In photographic film, it's Kodak and Fuji. In rent-a-cars, it's Hertz and Avis. In mouthwash, it's Listerine and Scope. In hamburgers, it's McDonald's and Burger King. In sneakers, it's Nike and Reebok, In toothpaste, it's Crest and Colgate.

- Back in 1969, there were 3 major brands of a certain product. The leader had about 60 percent of the market, the no. 2 brand had a 25 percent share, and the no. 3 brand has a 6 percent share. The rest of the market included either private label or minor brands. The law of duality suggests that these market shares are unstable. Furthermore, the law predicts that the leader will lose market share and No. 2 will gain.
- 22 years later, the leader dropped down to 45 percent of the market. No. 2 brand has 40 percent, and no. 3 has 3 percent. The products are Coca-cola, Pepsi-cola, and Royal Crown cola, respectively, but the principles apply to brands everywhere.

**Comment by quinnliu** [comment]
- When ur a weak No. 3, like Royal Crown, u aren't going to make much progress by going out & attacking the 2 strong leaders. What they could have done is carved out a profitable niche for themselves. The law of focus.

**Comment by quinnliu** [comment]
- Jack Welch, CEO of General Electric "only businesses that are No. 1 or No. 2 in their markets could win in the increasingly competitive global arena. In 32 of its 44 product categories in the US, P&G commands the No. 1 or No. 2 brands.

**Comment by quinnliu** [comment]
# The law of opposite: if ur shooting for 2nd place, ur strategy is determined by the leader
- if u want to establish a firm foothold on the 2nd rung of the ladder, study the firm above u. Where is it strong? and how do u turn that strength into a weakness?
- u must discover the essence of the leader and then present the prospect with the opposite. It's often the upstart versus old reliable.
- Coca-Cola is a 100 year old product. When u look at customers in a given product category, there seem to be 2 kinds of people. There are those who want to buy from the leader and there are those who don't want to buy from the leader. A potential No. 2 has to appeal to the latter group.

**Comment by quinnliu** [comment]
- As a product gets old, it often accrues some negative baggage. This is especially true in the medical field. Take aspirin, a product introduced in 1899. With thousands of medical studies conducted on aspirin, someone was bound to find flaws in the product. Sure enough, they found stomaach bleeding - just in time for the 1955 launch of Tylenol.

**Comment by quinnliu** [comment]
- Marketing is often a battle for legitimacy. The first brand that captures the concept is often able to protray its competitors as illegitimate pretenders.

**Comment by quinnliu** [comment]
# The law of division: over time, a category will divide and become 2+ categories

**Comment by quinnliu** [comment]
- Companies make a mistake when they try to take a well-known brand name in one category and use the same brand name in another category.
- Timing is also important. U can be too early to exploit a new category. Back in the fifties the Nash Rambler was America's first small car. But American Motors didn't have either the courage or the money to hang in there long enough for the category to develop.

**Comment by quinnliu** [comment]
# The law of perspective: marketing effects take place over an extended period of time. 
- Is alcohol a stimulant or a depressent?
- If u visit almost any bar & grill on Friday night after work, u'd swear that alcohol was a stimulant. the noise & laughter are strong evidence of alcohol's stimulating effects. 
- Chemically, alcohol is a strong depressent. But in the short term. by depressing a person's inhibitions, alcohol acts like a stimulant

**Comment by quinnliu** [comment]
- Does a sale increase a company's business or decrease it? Obviously, in the short term, a sale increases business. But there's more and more evidence to show that sales decrease business in the long term by educating customers not to buy at "regular" prices. 
- A sale says ur regular prices are too high. After the sale is over, customers tend to avoid a store with a "sale" reputation.
- the largest furniture company in the NYC area, Seamans, has been running a sale every week. Recently, Seamans went bankrupt.

**Comment by quinnliu** [comment]
- book published = January 13, 1993

**Comment by quinnliu** [comment]
- At first, the donald was successful. then he branched out and put his name on anything the banks would lend him money for. What's a trump? a hotel, 3 casinos, 2 condominiums, 1 airline, 1 shopping center. 
- today trump is $1.4 billion in debt. what made him successful in the short term is exactly what caused him to fail in the long term. line extension.

**Comment by quinnliu** [comment]
- if violating any of our laws was a punishable offense, a large portion of corporate America would be in jail. 
- by far the most violated law in our book is the law of line extension
- take IBM. years ago when IBM was focused on mainframe computers, the company made a ton of money. today IBM is into everything and barely breaking even. In 1991, for example, IBM's revenues were $65 billion. Yet the company would up losing $2.8 billion. That's almost $8 million a day.
- In addition to selling mainframe computers, IBM markets personal computers, pen computers, workstations, midrange computers, software, networks, telephones, u name it. IBM even tried to get into the home computer market with the PCjr.

**Comment by quinnliu** [comment]
- But marketing is a battle of perception, not product. In the mind, A-1 is not the brand name, but the steak sause itself. 
- In spite of an $18 million advertising budget, the A-1 poultry launch was a dismal failure.

**Comment by quinnliu** [comment]
- Back in 1978, when 7-up was simply the lemon-lime uncola, it had a 5.7 percent share of the soft-drink market. Then the company added 7-up gold, cherry 7-up, and assorted diet versions. Today 7-up's share is down to 2.5 percent.

**Comment by quinnliu** [comment]
- paused at pg. 72

**Comment by quinnliu** [comment]
- one day a company is tightly focused on a single product that is highly profitable. The next day the same company is spread thin over many products and is losing money. 
- take IBM. years ago when IBM was focused on mainframe computers, the company made a ton of money. Today IBM is into everything and barely breaking even. In 1991, for example, IBM's revenues were $65 billion. Yet the company would up losing $2.8 billion. That's almost $8 million a day. In addition to selling mainframe computers, IBM markets personal computers, pen computers, workstations, midrange computers, software, networks, telephones, u name it. IBM even tried to get into the home computer market with the PCjr.

**Comment by quinnliu** [comment]
- when a company becomes incredibly successful, it invariably plants the seeds for its future problems. 
- Microsoft Corp. said it is aggressively seeking the dominant share in every major software applications category in the personal computer field.
- suggested that Microsoft might be able to achieve as much as 70 percent share in every major applications category.

**Comment by quinnliu** [comment]
- Invariably, the leader in any category is the brand that is not line extended. Take baby food, for example, Gerber has 72 percent of the market, way ahead of Beech-Nut and Heinz, the two line-extended brands. 
- In spite of evidence that line extensions don't work, companies continue to pump them out. Here are some examples:

- Ivory soap. Ivory shampoo?
- Life Savers candy. Life savers gum?
- Chanel. Chanel for men?
- Heinz ketchup. Heinz baby food?
- Adidas running shoes. Adidas cologne?
- Levi's blue jeans. Levi's shoes?

**Comment by quinnliu** [comment]
- Less is more. If u want to be successful today, u have to narrow the focus in order to build a position in the prospect's mind. 
- What does IBM stand for? It used to stand for "mainframe computers". Today it stands for everything, which means it stands for nothing.

**Comment by quinnliu** [comment]
- The antidote for line extension is corporate courage, a commodity in short supply.

**Comment by quinnliu** [comment]
# The law of sacrifice: u have to give up something in order to get something
- The law of sacrifice is the opposite of the law of line extension. If u want to be successful today, u should give something up.
- There are three things to sacrifice: product line, target market, and constant change

**Comment by quinnliu** [comment]
- where is it written that the more u have to sell, the more u sell?
- the full line is a luxury for a loser. If u want to be successful, u have to reduce ur product line, not expand it. 
- Take Emery Air Frieght. Emery was in the air freight services business. Anything u wanted to ship u could ship via Emery. Small packages, large packages, overnight service, delayed service. From a marketing pov, what did Federal express do? It concentrated on one service: small packages overnight. Today Federal express is a much bigger company than Emery.

**Comment by quinnliu** [comment]
- Then what did Federal Express do? The company did the same thing Emery did. It threw away its overnight position by buying Tiger International's Flying Tiger cargo line for $880 million. Now Federal Express is a worldwide air cargo company without a worldwide position. In just 21 months Federal Express lost $1.1 billion in its international operations.

**Comment by quinnliu** [comment]
- It's bad enough that Federal Express walked away from the "overnight" idea. What's worse is that it didn't replace the idea with a new one. 
- Eveready was the long-time leader in batteries. 
- Then P.R. Mallory introduced a line of alkaline batteries only. The company gave the line a better name: Duracell. 
- The power of the sacrifice for Duracell was in being able to put the "long-lasting battery" idea in the mind of the prospect. Duracell lasts twice as long as Eveready, said the advertising.

**Comment by quinnliu** [comment]
- The world of business is populated by big, highly diversified generalists and small, narrowly focused specialists. If line extension and diversification were effective marketing strategies, u'd expect to see the generalists riding high. But they're not. Most of them are in trouble.
- The generalist is weak. Take Kraft, for example. Everybody thinks Kraft is a strong brand name. In jellies and jams, Kraft has 9 percent of the market. But Smucker's has 35 percent. Kraft means everything, but with a name like Smucker's it has to be jelly or jam because that's all Smucker's makes. 
- In mayonnaise, Kraft has 18 percent of the market. But Hellmann's has 42 percent. 
- Take the retail industry. Which retailers are in trouble today? The department stores. And what's a department store? A place that sells everything. That's a recipe for disaster.
- Campeau, L.J. Hooker, and Gimbels all wound up in bankruptcy court. Ames department stores filed for bankruptcy. Hills department stores filed for bankruptcy. Macy's the owner of the world's largest store, filed for bankruptcy.

**Comment by quinnliu** [comment]
- Interstate Department Stores also went bankrupt. So the company looked at the books and decided to focus on the only product it made money on: toys. As long as Interstate was going to focus on toys, it decided to change its name to Toys "R" Us. Today Toys "R" Us does 20 percent of the retail toy business in the country. Very profitably, too. In it's last fiscal year, Toys "R" Us made $326 million on sales of $5.5 billion.

**Comment by quinnliu** [comment]
- In the retail field generally, the big successes are the specialists:
- The Limited. Upscale clothing for working women
- The gap. casual clothing for the young at heart.
- Benetton. Wool & cotten clothing for young swingers.
- Victoria's Secret. Sexy undergarments.
- Foot Locker. Athletic shoes
- Banana Republic. Safari wear.

**Comment by quinnliu** [comment]
- The second sacrifice, target market. Where is it written that u have to appeal to everybody?
- Take the cola field. Coca-Cola got into the prospect's mind first and built a powerful position. In the late fifties, for example, Coke outsold Pepsi more than five to one. What could Pepsi-Cola do to go against Coke's powerful position?
- In the early sixties Pepsi-Cola finally developed a strategy based on the concept of sacrifice. The company sacrificed everything except the teenage market. Then it brilliantly exploited this market by hiring it's icons: Michael Jackson, Lionel Richie, Don Johnson. 
- Within one generation, Pepsi closed the gap. Today it is only 10 percent behind Coca-Cola in total US cola sales.

**Comment by quinnliu** [comment]
There seems to be an almost religious belief that the wider net catches more customers, in spite of many examples to the contrary.

**Comment by quinnliu** [comment]
- Look at cigarette advertisements, especially old cigarette ads. They invariably show both a man & a woman. Why? In an age when most smokers were men, cigarette manufacturers wanted to broaden their market. We got the men, let's go out and get the women, too. 
- So what did Philip Morris do? It narrowed the focus to men only. And then it narrowed the focus even more to a man's man, the cowboy. Today, Marlboro is the largest-selling cigarette in the world. In the US, Marlboro is the largest-selling cigarette amoung men & women.

**Comment by quinnliu** [comment]
- The target is not the market. That is, the apparent target of ur marketing is not the same as the people who will actually buy ur product. Even though Pepsi-Cola's target was the teenager, the market was everybody. The 50-year-old guy who wants to think he's 29 will drink the Pepsi.

**Comment by quinnliu** [comment]
- Finally, the third sacrifice: constant change. Where is it written that u have to change ur strategy every year at budget review time?
- If u try to follow the twists and turns of the market, ur bound to wind up off the road. The best way to maintain a consistent position is not to change it in the first place.

**Comment by quinnliu** [comment]
- People Express had a brillian "narrow" position to start with. It was the no-frills airline that flew to no-frills cities at no-frills prices. People used to get on a People Express plane and say, "Where are we going?" They didn't care, as long as it was cheap enough.
- What did People Express do after it became successful? It tried to be all things to all people. It invested in new equipment, like 747s. It started to fly the heavily traveled routes to places like Chicago and Denver, not to mention Europe. It bought Frontier Airlines. It added frills, like first-class sections.

**Comment by quinnliu** [comment]
- White Castle, on the other hand, has never changed its position. A white castle today not only looks the same as a white castle did 60 years ago, it also sells the same "frozen sliders" as unbelievably low prices. Would u believe the average White Castle does more than $1 million a year in revenues? Good things come to those who sacrifice.

**Comment by quinnliu** [comment]
# The Law of Attributes
- for every attribute, there is an opposite, effective attribute
- in chapter 6 the law of exclusivity we made the point that u can't own the same word or position that ur competitor owns. u must find ur own word to won. u must seek out another attribute. 
- too often a company attempts to emulate the leader. they must know what works, goes the rationale, so let's do something similar. not good thinking.
- it's much better to search for an opposite attribute that will allow u to play off against the leader. the key word here is opposite-similar won't do.
- Coca-cola was the original and thus the choice of older people. Pepsi successfully positioned itself as the choice of the younger generation. 
- since crest owned cavities, other toothpastes avoided cavities and jumped on other attributes like taste, whitening, breath protection, and more recently, baking soda. 
- marketing is a battle of ideas. so if ur to succeed, u must have an idea or attribute of ur own to focus ur efforts around. without one, u had better have a low price, a very low price.

**Comment by quinnliu** [comment]
- some say all attributes are not created equal. Some attributes are more important to customers than others. u must try and own the most important attribute. Cavity prevention is the most important attribute in toothpaste. It's the one to won. But the law of exclusivity points to the simple truth that once an attribute is successfully taken by ur competition, it's gone. U must move on to a lesser attribute and live with a smaller share of the category. Ur job is to seize a different attribute, dramatize the value of ur attribute, and thus increase ur share.

**Comment by quinnliu** [comment]
- A company that never laughs at new attributes that are exactly the opposite of their current products is Gillette, the world's No. 1 razor blade maker. Its dominance revolves around its high-technology razors and cartridge systems. When an upstart from France brought an opposite attribute to the category in the form of a "disposable" razor, Gillette could have laughed and wheeled out its research on how America wants hefty, expensive, high-technology razors. But it didn't. Instead, Gillette jumped in with a disposable razor of its own, called Good News. By spending heavily, Gillette was able to win the battle of the disposables.
- Today the Gillette Good News razor dominates the disposable category, which has grown to dominate the razor blade business. Moral: U can't predict the size of a new attribute's share, so never laugh

**Comment by quinnliu** [comment]
- To drive the concept into prospects' minds, Burger King would need a term. It could be grow up. Grow up to the flame-broiled taste of Burger King. 
- The new concept for Burger King would strike fear and terror in the boardroom at McDonald's, always a good sign of an effective program.

**Comment by quinnliu** [comment]
# The Law of Candor: when u admit a negative, the prospect will give u a positive
- Listerine = the taste u hate twice a day

**Comment by quinnliu** [comment]
- So it may come as a surprise to u that one of the most effective ways to get into a prospect's mind is to first admit a negative and then twist it into a positive. 
- Avis is only No. 2 in rent-a-cars
- with a name like Smucker's, it has to be good
- the 1970 VW will stay ugly longer
- Joy. The most expensive perfume in the world.
- Why does a dose of honesty work so well in the marketing process?
- First and foremost, candor is very disarming. Every negative statement u make about urself is instantly accepted as truth. positive statements, on the other hand, are looked at as dubious at best. Especially in an advertisement.

**Comment by quinnliu** [comment]
- u have to prove a positive statement to the prospect's satisfaction. No proof is needed for a negative statement. 
- the 1970 VW will stay ugly longer. A car that ugly must be reliable, thinks the prospect. 
- joy. the most expensive perfume in the world. if people are willing to pay $375 an ounce, it must be a sensational perfume. 
- with a name like Smucker's it has to be good. Most companies, especially family companies, would never make fun of their own name. Yet the Smucker family did, which is one reason why Smucker's is the No. 1 brand of jams and jellies. If ur name is bad, u have 2 choices, change the name or make fun of it.

**Comment by quinnliu** [comment]
- positive thinking has been highly overrated. The explosive growth of communications in our society has made people defensive and cautious about companies trying to sell them anything. Admitting a problem is something that very few companies do. 
- When a company starts a message by admitting a problem, people tend to, almost instinctively, open their minds. 
- Think about the times that someone came to u with a problem and how quickly u got involved and wanted to help. Now think about people starting off a conversation about some wonderful things they are doing. u probably were a lot less interested.

**Comment by quinnliu** [comment]
- Not only did the company admit the product tasted bad, it admitted that people actually hated it. This set up the selling idea that Listerine "kills a lot of germs". 
- one final note: the law of candor must be used carefully and with great skill. First ur "negative" must be widely perceived as a negative. It has to trigger an instant agreement with ur prospect's mind. If the negative doesn't register quickly, ur prospect will be confused and will wonder, "what's this all about?"
- Next u have to shift quickly to the positive. The purpose of candor isn't to apologize. The purpose of candor is to set up a benefit that will convince ur prospect. This law only proves the old maxim: Honesty is the best policy.

**Comment by quinnliu** [comment]
# The Law of Singularity: in each situation, only one move will produce substantial results
- History teaches that the only thing that works in marketing is the single, bold stroke. Furthermore, in any given situation there is only one move that will produce substantial results.

**Comment by quinnliu** [comment]
- Military strategist and author B.H. Liddell Hart calls this bold stroke "the line of, least expectation" - The allied invasion came at Normandy, a place whose tide and rocky shore the Germans felt would be an unlikely choice for a landing of any scale. 
- So it is in marketing. Most often there is only 1 place where a competitor is vulnerable. And that place should be the focus of the entire invaliding force.

**Comment by quinnliu** [comment]
- what works in marketing is the same as what works in the military: the unexpected. 
- Hannibal came over the Alps, a route deemed impossible to scale. Hitler came around the Maginot line and sent his panzer divisions through the Ardennes, terrain the French generals thought impossible to traverse with tanks. He did it twice- once in the Battle of France and again in the Battle of the Bulge. 
- In recent years there have been only 2 strong moves made against GM. Both were flanking moves around the GM Maginot Line. The japanese came at the low end with small cars like Toyota, Datsun, and Honda. The Germans came at the high end with super-premium cars like Mercedes and BMW. 
- With the success of Japanese and German flanking attacks, General Motors was under pressure to commit resources in an attempt to shore up the bottom and the top of its lines. Cadillacs were too cheap to block the high-priced German imports.

**Comment by quinnliu** [comment]
- First of all, Coke has to bite the bullet and drop New Coke. Not because it's a loser or an embarrassment, but because the existence of New Coke blocks the company from effectively using the only weapon it has. With new Coke safely tucked away in the archives, Coke would be able to invoke the law of focus and bring back the concept of the "Real Thing" and use it against Pepsi. 
- to pull the trigger, Coke could go on television and say to the Pepsi generation, all right kids, we're not going to push u. when ur ready for  the read thing, we've got it for u. that would be the beginning of the end of the pepsi generation.

**Comment by quinnliu** [comment]
- to find that singular idea or concept, marketing managers have to know what's happening in the marketplace. they have to be down at the front in the mud of the battle. they have to know what's working and what isn't. they have to be involved.

**Comment by quinnliu** [comment]
# The Law of Unpredictability: unless u write ur competitor's plans, u can't predict the future

**Comment by quinnliu** [comment]
- Most companies live from quarterly report to quarterly report. That's a recipe for problems. Companies that live by the numbers, die by the numbers. 
- Monaghan couldn't own the words home delivery until he had enough franchises to afford national advertising. He accomplished both objectives, and today Domino's is a $2.65 billion company with a 4 percent share of the home delivery business

**Comment by quinnliu** [comment]
- One final note that's worth mentioning: there's a difference between "predicting" the future and "taking a chance" on the future. Orville Redenbacher's Gourmet popping corn took a chance that people would pay twice as much for a high-end popcorn. Not a bad risk in today's affluent society. 
- no one can predict the future with any degree of certainty. nor should marketing plans try to.

**Comment by quinnliu** [comment]
# The law of success: success often leads to arrogance and arrogance to failure
- ego is the enemy of successful marketing
- objectivity is what's needed
- Donald Trump & Robert Maxwell are two examples of people blinded by early success and untainted by humility. and when ur blind, it is indeed hard to focus. 
- Mr. Trump's strategy was to put his name on everything, committing the cardinal sin of line extension.
- Success is often the fatal element behind the rash of line extensions. When a brand is successful, the company assumes the name is the primary reason for the brand's success. So they promptly look for other products to plaster the name on. 
- Actually it's the opposite. The name didn't make the brand famous (although a bad name might keep the brand from becoming famous). The brand got famous because u made the right marketing moves. in other words, the steps u took were in tune with the fundamental laws of marketing.

**Comment by quinnliu** [comment]
- Actually ego is helpful. It can be an effective driving force in building a business. What hurts is injecting ur ego in the marketing process. Brilliant marketers have the ability to think like a prospect thinks. They put themselves in the shoes of their customers. They don't impose their own view of the world on the situation. Keep in mind that the world is all perception anyway, and the only thing that counts in marketing is the customer's perception.

**Comment by quinnliu** [comment]
- as their successes mounted, companies like General motors, sears, roebuck, and IBM became arrogant. They felt they could do anything they wanted to in the marketplace. success leads to failure. 
- consider Digital Equipment Corporation, the company that brought us the minicomputer. Starting from scratch, DEC became an enormously successful $14 billion company. 
- DEC's founder is Kenneth Olsen. His success made Ken such a believer in his own view of the computer world that he pooh-poohed the personal computer, then open systems, and, finally, reduced instruction set computing (RISC). In other words, Ken Olsen ignored three of the biggest developments in the computer category. 
- the bigger the company, the more likely it is that the chief executive has lost touch with the front lines

**Comment by quinnliu** [comment]
- the bigger the company, the more likely it is that the chief executive has lost touch with the front lines. this might be the single most important factor limiting the growth of a corporation. all other factors favor size. marketing is war, and the first principle of warfare is the principle of force. the larger army, the larger company, has the advantage.
- but the larger company gives up some  of that advantage if it cannot keep itself focused on the marketing battle that takes place in the mind of the customer.
- the shootout at general motors between roger smith & ross perot illustrates the point. when he was on the GM board, Ross perot spent his weekends visiting dealers and buying cars. he was critical of roger smith for not doing the same.

**Comment by quinnliu** [comment]
- if ur a busy CEO, how do u gather objective information on what is really happening? how do u get around the propensity of middle management to tell u what they think u want to hear?

**Comment by quinnliu** [comment]
# The law of failure: faillure is to be expected and accepted
- too many companies try to fix things rather than drop things. let's reorganize to save the situation is their way of life. 
- admitting a mistake and not doing anything about it is bad for ur career. A better strategy is to recognize failure early and cut ur losses. American motors should have abandoned passenger cars and focused on Jeep. IBM should have dropped copiers and Xerox should have dropped computers years before they finally recognized their mistakes.

**Comment by quinnliu** [comment]
- the japanese seem to be able to admit a mistake early and then make the necessary changes. their consensus management style tends to eliminate the ego. since a large number of people have a small piece of a big decision, there is no stigma that can be considered career damaging. in other words, its a lot easer to live with "we were all wrong" then the devastating "i was wrong". this egoless approach is a major factor in making the Japanese such relentless marketers. It's not that they don't make mistakes, but when they do, they admit them, fix them, and just keep coming.

**Comment by quinnliu** [comment]
- this leads to a failure to take risks. it's hard to be first in a new category without sticking ur neck out. when the senior executive has a high salary and a short time to retirement, a bold move is highly unlikely.

**Comment by quinnliu** [comment]
- in some american companies nothing gets done unless it benefits the personal agenda of someone in top management. this severely limits the potential marketing moves a company can make. an idea gets rejected not because it isn't fundamentally sound but because no one in top management will personally benefit from its success.

**Comment by quinnliu** [comment]
# The Law of Hype: the situation is often the opposite of the way it appears in the press

**Comment by quinnliu** [comment]
- When IBM was successful, the company said very little. Now it throws a lot of press conferences. 
- when things are going well, a company doesn't need the hype. when u need the hype, it usually means ur in trouble. 
- the essence of the hype was not just that the new product was going to be successful. the essence of the hype was that existing products would not be obsolete.
- did anyone predict the overthrow of communism and the Soviet Union? not really. it was only after the process had started that the press jumped on the "crumbling communist empire" story.

**Comment by quinnliu** [comment]
# 21 The law of acceleration: successful programs are not built on fads, they're built on trends

**Comment by quinnliu** [comment]
- When a fad disappears, a company often goes into a deep financial shock. What happened to Atari is typical in this respect. And look how Coleco Industries handled the Cabbage Patch Kids. Those homely dolls hit the market in 1983 and started to take off. Colesco's strategy was to milk the kids for all they were worth. 
- Hundreds of Cabbage Patch novelties flooded the toy stores. Pens, pencils, crayon boxes, games, clothing. 2 years later, Coleco racked up sales of $776 million and profits of $83 million. Then the bottom dropped out of the Cabbage Patch Kids. by 1988 Coleco went into Chapter 11.

**Comment by quinnliu** [comment]
- the most successful entertainers are the ones who control their appearances. they don't overextend themselves. they're not all over the place. they don't wear out their welcome.

**Comment by quinnliu** [comment]
- One way to maintain a long-term demand for ur product is to never totally satisfy the deman. But the best, most profitable thing to ride in mareting is a long-term trend.

**Comment by quinnliu** [comment]
# The law of resources: without adequate funding and idea won't get off the ground

**Comment by quinnliu** [comment]
- Some entrepreneurs see advertising as the solution to the problem of getting into prospects' minds. Advertising is expensive. It cost 9K/minute to fight WWII. It cost 22K/minute to fight the Vietnam War. A 1 min. commercial on the NFL super bowl will cost u 1.5 million. 
- Steve Jobs & Steve Wozniak had a great idea. But it was Mike Markkula's 91K that put Apple Computer on the map. For his money, Markkula got 1/3 of Apple.

**Comment by quinnliu** [comment]
- If u violate the immutable laws, u run the risk of failure. If u apply the immutable laws, u run the risk of being bad-mouthed, ignored, or even ostracized.
- have patience, the immutable laws of marketing will help u achieve success. and success is the best revenge of all.
